apt/minimization/minimizer.py

```diff
@@ -1027,8 +1023,11 @@ def _get_feature_to_remove(self, original_data, prepared_data, nodes, labels, fe
                     GeneralizeToRepresentative._remove_feature_from_cells(new_cells, cells_by_id, feature)
                     generalized = self._generalize_from_tree(original_data, prepared_data, nodes, new_cells,
                                                              cells_by_id)
-                    accuracy_gain = self.estimator.score(ArrayDataset(self.encoder.transform(generalized),
-                                                                      labels)) - current_accuracy
+                    if self.encoder:
```

Comment:
It seems like most of the code under "# divide by accuracy loss " is the same as that under the comment "# divide by accuracy gain", so could be extracted into a method and reused.